src/api-service/__app__/onefuzzlib/tasks/main.py

```diff
@@ -153,6 +153,23 @@ def get_by_task_id(cls, task_id: UUID) -> Union[Error, "Task"]:
         task = tasks[0]
         return task
 
+    @classmethod
+    def get_tasks_by_pool_name(cls, pool_name: str) -> Union[Error, List["Task"]]:
+        tasks = cls.search_states(states=TaskState.available())
+        if not tasks:
+            return Error(code=ErrorCode.INVALID_REQUEST, errors=["unable to find task"])
```

Comment:
Since the caller of this function turns Error into an empty list, you don't need to return an error. 

Removing `if not tasks: return Error(..)` makes this effectively the same.